vue-app/src/api/round.ts

```diff
@@ -54,23 +53,9 @@ export async function getCurrentRound(): Promise<string | null> {
   return fundingRoundAddress
 }
 
-async function getRoundNumber(roundAddress: string): Promise<number> {
```

Comment:
Not something we necessarily have to solve for here but noting in other clr.fund instances, we may actually want to track the round number for providing the ability to view past funding rounds (like on https://clr.fund/#/rounds).

Reply:
Oh okok, make sense. Will revert this part and will leave the function with a new subgraph query.

Reply:
Are you still planning to revert this? Or can this be solved in a different PR?